app.py

```diff
@@ -0,0 +1,166 @@
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from models import db , Profile
+from flask_bcrypt import Bcrypt
+from flask_jwt_extended import JWTManager, create_access_token, jwt_required, get_jwt_identity
+from flask_mail import Mail,Message
+from urllib.parse import unquote_plus, quote_plus
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///todos.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['MAIL_SERVER'] ='smtp.gmail.com'
```

Comment:
move all configurations into the environment and retrieve the values from there.